code/modules/surgery/organs/feature_organs/wings.dm

```diff
@@ -25,19 +25,15 @@
 /obj/item/organ/wings/anthro
 	name = "wild-kin wings"
 
-/obj/item/organ/wings/flight
-	unconditional_flight = TRUE
-	can_open = TRUE
-
-/obj/item/organ/wings/flight/angel
+/obj/item/organ/wings/aasimar
```

Comment:
And here. /obj/item/organ/wings/aasimar should be an unrelated object from /obj/item/organ/wings/flight. You don't need to delete  /obj/item/organ/wings/flight to add /obj/item/organ/wings/aasimar

Additionally you need to give it a name variable like `name = "aasimar wings"`